src/tests/unit/recipeService/getAllRecipes.test.ts

```diff
@@ -0,0 +1,38 @@
+import { saveRecipes } from '../../mocks/mockRecipe';
+import RecipeService from '../../../services/recipeService';
+import { setupTests } from '../../setupTests';
+
+const recipeService = new RecipeService();
+
+setupTests('getAllRecipes', () => {
+  describe('should return', () => {
+    test('existing recipes', async () => {
+      await saveRecipes(3);
+      const recipes = await recipeService.getAllRecipes();
+      expect(recipes).toHaveLength(3);
+    });
+
+    test('empty array if no recipes are present', async () => {
+      const recipes = await recipeService.getAllRecipes();
+      expect(recipes).toEqual([]);
+    });
+
+    test('paginated results', async () => {
+      await saveRecipes(20);
+      let recipes = await recipeService.getAllRecipes();
+      expect(recipes).toHaveLength(10);
+      recipes = await recipeService.getAllRecipes(undefined, 20);
+      expect(recipes).toHaveLength(20);
+      const pagedRecipes = await recipeService.getAllRecipes(2, 5);
+      expect(pagedRecipes[0]).toEqual(recipes[5]);
+    });
+
+    test('results sorted by descending creation time', async () => {
+      await saveRecipes(5);
+      const recipes = await recipeService.getAllRecipes();
+      expect(recipes[0].createdAt.getTime()).toBeGreaterThan(
+        recipes[1].createdAt.getTime()
+      );
```

Comment:
This does check only first and second item, what about the rest?

You could iterate over all returned recipes or use some library for testing this, e.g. https://www.npmjs.com/package/jest-sorted